### DISCUSSION TITLE
p.250 auth.NewJWTer関数呼び出しにおける引数の不一致

### DISCUSSION THREAD

**Discussion by ac0mz** [budougumi0617/go_todo_app#44] [Ideas]
書籍p.250（リスト20.32）のコードでは、
```go
jwter, err := auth.NewJWTer(rcli)
```
と記載されていますが定義済のシグネチャと一致せず、サンプルコード上では以下のように記述されていました。
```go
jwter, err := auth.NewJWTer(rcli, clocker)
```

また以下サンプルコード（section79）の時点まではNewMux関数はリファクタされていませんが、p.250（section84）では突然clockerを外側に宣言するよう修正されていました。
※こちらについては、auth.NewJWTerの引数にclockerを指定した記載があればリファクタの意味を理解できるだろうと思われますが、記載がなかったためclockerをどこから持ってくるか少々混乱しました。
https://github.com/budougumi0617/go_todo_app/blob/main/_chapter19/section79/mux.go

```go
// p.250以前
r := store.Repository{Clocker: clock.RealClocker{}}
```
```go
// p.250
clocker := clock.RealClocker{}
r := store.Repository{Clocker: clocker}
```

**Comment by kdnakt** [comment]
p.237のリスト20.17や、p.239のリスト20.19でも `NewJWTer(moq, c)` というClockerを第2引数にしていますが、p.233のリスト20.14のNewJWTer()関数初出時では引数が1つだけなので、ここも修正が必要そうです

**Comment by budougumi0617** [comment]
こちらも正誤表を更新しリリースタグをつけました。ご報告ありがとうございました。
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.5
https://github.com/budougumi0617/go_todo_app/blob/v1.0.5/errata.md
